app/Providers/AppServiceProvider.php

```diff
@@ -2,7 +2,10 @@
 
 namespace App\Providers;
 
+use Domain\Airline\Actions\CreateAirlineAction;
 use Illuminate\Support\ServiceProvider;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
```suggestion
use Illuminate\Support\ServiceProvider;
```
These classes are not being used.